pkg/accounting/accounting.go

```diff
@@ -60,6 +60,7 @@ type Options struct {
 	Logger           logging.Logger
 	Store            storage.StateStorer
 	Settlement       settlement.Interface
+	Pricing          pricing.Interface
```

Comment:
I would move all fields out of the Options struct that are not really optional.

Reply:
removed